pipreqs/pipreqs.py

```diff
@@ -155,56 +151,48 @@ def filter_line(line):
 
 def generate_requirements_file(path, imports, symbol):
     with _open(path, "w") as out_file:
-        logging.debug('Writing {num} requirements: {imports} to {file}'.format(
-            num=len(imports),
-            file=path,
-            imports=", ".join([x['name'] for x in imports])
-        ))
-        fmt = '{name}' + symbol + '{version}'
-        out_file.write('\n'.join(
-            fmt.format(**item) if item['version'] else '{name}'.format(**item)
-            for item in imports) + '\n')
+        logging.debug(
+            "Writing {num} requirements: {imports} to {file}".format(
+                num=len(imports), file=path, imports=", ".join([x["name"] for x in imports])
+            )
+        )
+        fmt = "{name}" + symbol + "{version}"
+        out_file.write(
```

Comment:
please stop changing formatting and quotes convention on every commit you guys do haha